com/paypal/Perfection.java

```diff
@@ -0,0 +1,42 @@
+package com.paypal;
+
+public class Perfection {
```

Comment:
Can use a final class and private constructor for that the class doesn't allow extends and intantiation